content/en/docs/setup/getting-started/index.md

```diff
@@ -348,6 +348,30 @@ by viewing the Bookinfo product page using a browser.
 
 1.  Paste the output from the previous command into your web browser and confirm that the Bookinfo product page is displayed.
 
+### Apply default destination rules
+
+Before you can use Istio to control the Bookinfo version routing, you need to define the available
+versions, called *subsets*, in [destination rules](/docs/concepts/traffic-management/#destination-rules).
+
+Run the following command to create default destination rules for the Bookinfo services:
+
+{{< text bash >}}
+$ kubectl apply -f @samples/bookinfo/networking/destination-rule-all.yaml@
+{{< /text >}}
+
+{{< tip >}}
+The `default` and `demo` [configuration profiles](/docs/setup/additional-setup/config-profiles/) have [auto mutual TLS](/docs/tasks/security/authentication/authn-policy/#auto-mutual-tls) enabled by default.
+To enforce mutual TLS, use the destination rules in `samples/bookinfo/networking/destination-rule-all-mtls.yaml`.
+{{< /tip >}}
+
+Wait a few seconds for the destination rules to propagate.
+
+You can display the destination rules with the following command:
+
+{{< text bash >}}
+$ kubectl get destinationrules -o yaml
+{{< /text >}}
+
```

Comment:
While I'm not necessarily keen on adding this section, it does make sense if we have the `Next steps` section at the bottom where the user may follow a link which and their Istio environment may not be correct (those pages are written to follow the `BookInfo` page from which this section is borrowed). I'm not sure how many of the follow-on pages would need updating (similar to what was added to request-routing in this commit), but at quick glance, maybe 3.

Reply:
Rather than copying the whole section here, why not just fix the problem by correcting the tip in the bookinfo example: `If you installed Istio using the Getting Started instructions, you already have Bookinfo installed and you can skip most of these steps and proceed straight to Apply default destination rules(link).`

Reply:
Thanks for the suggestions. I think revising the tip section makes sense.